R/methods.R

```diff
@@ -40,6 +29,22 @@ format.lint <- function(x, ..., width = getOption("lintr.format_width")) {
   )
 }
 
+build_line_ref <- function(x) {
+  line_ref <- paste0(
+    x$filename, ":",
+    as.character(x$line_number), ":",
+    as.character(x$column_number)
+  )
+
+  create_clickable_link <- cli::ansi_has_hyperlink_support()
+  if (create_clickable_link) {
+    line_ref <- paste0("{.path ", line_ref, "}")
+    line_ref <- cli::format_inline(line_ref)
```

Comment:
```suggestion
    line_ref <- cli::format_inline("{.path {line_ref}}")
```

Actually, this would be sufficient!